samples/go/demo-server/web_server.go

```diff
@@ -152,3 +155,10 @@ func notFound(w http.ResponseWriter, r *http.Request) {
 	newParams = append(newParams, params...)
 	hndl(w, r, newParams)
 }
+
+func handleBaseGet(w http.ResponseWriter, req *http.Request, params httprouter.Params) {
+	d.PanicIfTrue(req.Method != "GET", "Expected get method.")
+
+	fmt.Fprintf(w, nomsBaseHtml)
```

Comment:
should these be reversed? set/send header first then send html to responseWriter? I am guessing maybe responseWriter buffers the Fprintf so as to be able to send appropriate headers before sending the output.